pages/index.js

```diff
@@ -1,3 +1,24 @@
+import ListItemSquare from "../components/ListItemSquare";
+import useSWR from "swr";
+import { fetcher } from "../helpers/api";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  const { data: movies, error } = useSWR("/api/movies", fetcher);
+
+  if (!movies) {
+    return <h2> loading..</h2>;
+  }
+
+  if (error) return <div>failed to load</div>;
+
+  return (
+    <>
+      <main>
+        <h1>series & movies</h1>
+        <ul>
+          <ListItemSquare movies={movies} />
+        </ul>
+      </main>
+    </>
+  );
```

Comment:
You don't need the react fragments here, because the `<main>` element groups the nested children. 
For readability and logical reasons I would recommend to render the `<ul>` inside the `<ListItemSquare>` -component. By the way I suggest to rename this component into `<MovieList>` or `<SeriesMovieList>` or something similar. ✌️

```suggestion
  return (
      <main>
        <h1>series & movies</h1>
          <ListItemSquare movies={movies} />
      </main>
  );
```